src/act.offensive.cpp

```diff
@@ -598,7 +598,19 @@ bool passed_flee_success_check(struct char_data *ch) {
   if (!can_hurt(ch, FIGHTING(ch), TRUE, 0))
     return TRUE;
 
-  return success_test(GET_QUI(ch), GET_QUI(FIGHTING(ch))) > 0;
+  if success_test(GET_QUI(ch), GET_QUI(FIGHTING(ch)) + racial_flee_modifier) > 0
```

Comment:
If-statements need to have their conditions enclosed in parens like `if (x)`, and almost always should be written as block statements:
```c++
if (x) {
  your code here
}```